src/meta/load_balance_policy.cpp

```diff
@@ -128,7 +131,9 @@ generate_balancer_request(const app_mapper &apps,
                           const partition_configuration &pc,
                           const balance_type &type,
                           const rpc_address &from,
-                          const rpc_address &to)
+                          const rpc_address &to,
+                          const host_port &hp_from,
+                          const host_port &hp_to)
```

Comment:
Is it enough to pass just hp_from and hp_to? The ip addresses will be resolved internally.

Reply:
That need another parameter `dns_resolver`